src/plugins/node-workspace.ts

```diff
@@ -40,7 +40,10 @@ export default class NodeWorkspaceDependencyUpdates extends ManifestPlugin {
         for (const [path, fileData] of pkg.prData.changes) {
           if (path === `${pkg.config.path}/package.json`) {
             this.log(`found ${path} in changes`, CheckpointType.Success);
-            pathPkgs.set(path, JSON.parse(fileData.content!) as PackageJson);
```

Comment:
maybe

```typescript 
if (this.indentContent === undefined) {
  this.indentContent = fileData.content;
}
pathPkgs.set(path, JSON.parse(fileData.content!) as PackageJson);
```

Reply:
I understood, thank you! I'll make the changes soon.